wpilibj/src/main/java/edu/wpi/first/wpilibj/command/Command.java

```diff
@@ -43,47 +42,56 @@
  * @see IllegalUseOfCommandException
  */
 public abstract class Command implements NamedSendable {
-
   /**
    * The name of this command.
    */
   private String m_name;
+
   /**
    * The time since this command was initialized.
    */
   private double m_startTime = -1;
+
   /**
    * The time (in seconds) before this command "times out" (or -1 if no timeout).
    */
   private double m_timeout = -1;
+
   /**
    * Whether or not this command has been initialized.
    */
   private boolean m_initialized = false;
+
   /**
    * The requirements (or null if no requirements).
```

Comment:
Please remove the `(or null if no requirements)` because that is no longer true.  It now is just an empty set.